pyblish_nuke/lib.py

```diff
@@ -60,7 +60,8 @@ def show():
 
     """
 
-    window = (_discover_gui() or _show_no_gui)()
+    parent = QtGui.QApplication.activeWindow()
```

Comment:
This may not be safe if window other than the main window is active at the time of showing it. It wouldn't be an issue when triggering it from the menu (assuming you can't tear it off like in Maya, as then it would become a window and hence the parent of Pyblish). But when showing it via code would have it parented to whatever is active at the time of the code being run.

Also don't forget that QtGui is only valid in Qt 4. We should update Qt.py to warn against it.

Something like this should cover it.

```python
from Qt import QtWidgets

current = QtWidgets.QApplication.activeWindow()
while current:
    parent = current
    current = parent.parent()

print(parent)
```